main.py

```diff
@@ -22,8 +24,12 @@ def greet():
         profile = ChromeProfile(email[0], email[1], email[2])
         driver = profile.retrieve_driver()
         profile.start()
+        tera.login(driver)
+        sleep(10)
```

Comment:
Sleeping time should be a constant for easier tweak, This sleep does not belong to this part